packages/eslint-plugin/tests/docs.test.ts

```diff
@@ -85,9 +85,14 @@ describe('Validating rule metadata', () => {
         // not perfect but should be good enough
         const ruleFileContents = fs.readFileSync(
           path.resolve(__dirname, `../src/rules/${ruleName}.ts`),
+          'utf-8',
         );
 
-        expect(ruleFileContents.includes('getParserServices')).toEqual(
+        const requiresTypeChecking = /getParserServices(\(\s*[a-z]+\s*)\)/.test(
+          ruleFileContents,
+        );
```

Comment:
this should also catch `getParserServices(context, true)`

Reply:
it should also catch
```
getParserServices(context, false)
```

but yes, added :>